python/meterpreter/meterpreter.py

```diff
@@ -483,7 +483,7 @@ def get_system_arch():
         ctypes.windll.kernel32.GetNativeSystemInfo(ctypes.byref(sysinfo))
         values = {0:'x86', 5:'armle', 6:'IA64', 9:'x64'}
         arch = values.get(sysinfo.wProcessorArchitecture, uname_info[4])
-    if arch == 'x86_64':
+    if arch == 'x86_64' or arch == 'AMD64':
```

Comment:
Do you think this should be a case insensitive test instead? I'm not sure about the exact output of `platform.uname()`.

Reply:
okay, I'll update it, thanks!